.eslintrc.json

```diff
@@ -0,0 +1,97 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "extends": [
+    "plugin:react/recommended",
+    "airbnb",
+    "airbnb-typescript/base",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:react/jsx-runtime",
+    "prettier",
+    "plugin:cypress/recommended"
+  ],
+  "overrides": [],
+  "parserOptions": {
+    "ecmaVersion": "latest",
+    "sourceType": "module",
+    "project": "./tsconfig.json",
+    "extraFileExtensions": [".scss"]
+  },
+  "ignorePatterns": ["*.config.[jt]s", "/dist/*"],
+  "plugins": ["react", "import"],
+  "rules": {
+    "react/jsx-filename-extension": [2, { "extensions": [".js", ".jsx", ".ts", ".tsx"] }],
+    "react/jsx-no-bind": [0],
+    "react/require-default-props": [0],
+    "react/function-component-definition": [0],
+    "import/no-absolute-path": "off",
```

Comment:
It won't let me add add absolute paths to **/images/** imports, so added turned this off but not good practice - need to find another solution